packages/ui-proxy/bin/ui-proxy.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const args = require('minimist')(process.argv.slice(2));
+
+const {
+    DEFAULT_PROXY_PORT,
+    DEFAULT_STATIC_PORT,
+    DEFAULT_REPLACE_SERVER,
+    DEFAULT_LIVESYNC_PORT
+} = require('../src/constants');
+const startProxy = require('../src/start-proxy');
+
+if (!args.buildPath) {
+    console.error('Argument "buildPath" is required, please see readme.md');
```

Comment:
The *parameter* is named `build-path`, not `buildPath`.

Reply:
Also, if this parameter is *that necessary*, let's make it more user-friendly though, e.g. so user could call it as
```
ui-proxy ./src
```
I mean, let's make it the first argument of the command.